mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -5539,7 +5541,7 @@ fn test_peer_time_sync_validation() {
     };
 
     // check with small time offset:
-    let (bob_peers, alice_peers) = start_peers_with_time_offset(timeoffset_tolerable);
+    let (bob_peers, alice_peers) = start_peers_with_time_offset(timeoffset_tolerable, false);
```

Comment:
why don't we run the seednode check here?

Reply:
'false' is to set `"skip_seednodes_check"=false`,  so actually means 'do seednode check'.

Reply:
oh my bad